### ISSUE TITLE
Document the design for beta.nsf.gov/Drupal login using NSF ADFS authentication service

### ISSUE THREAD

**Issue by mtboesch** [18F/nsf#140]
## User story
As a technical team member, I want to understand and be able to share with NSF DIS how we are going to SSO log in on beta.nsf.gov/Drupal using the NSF ADFS so that we can meet ATO requirements and ensure compliance.

## Definition of done
- [ ] Design is documented and required modules and configurations are identified 
- [ ]  Implement a POC on cloud.gov
- [ ] Design reviewed by NSF DAS and they have no further questions

**Comment by mtboesch** [comment]
Will we need to use SimpleSAMLphp and the SimpleSAMLphp Authentication Drupal module for this?  I can setup meetings with 18F and the NSF IAM team to exchange technical details.

**Comment by fureigh** [comment]
@mtboesch Yes, I believe so. Can you arrange test credentials for development? Let's talk on the 22nd about whether the documentation needs to be in a specific format, etc.
